src/wrapper/simulation_objects.py

```diff
@@ -35,6 +36,28 @@ def add_subscriptions(self):
         """
         raise NotImplementedError()
 
+    @staticmethod
+    @abstractmethod
+    def from_simulation(
+        simulation_object, updater: "SimulationObjectUpdatingComponent"
+    ) -> "SimulationObject":
+        """This method is called to initialize the object from the simulator.
+        When using SUMO, the simulation will not be started when this method is called.
+
+        :param simulation_object: The simulation object to initialize this object from.
+        """
+        raise NotImplementedError()
+
+    @abstractmethod
+    def from_running_simulation(self) -> None:
```

Comment:
It's just a tiny thing, but I thought `from_running_simulation` is another constructor additional to `from_simulation` 😄

Reply:
Do you have a better name?

Reply:
Something like `add_all_connections` or `add_all_connections_to_simulation_objects` if I understand the method right 😄 It's not a big thing